### PR TITLE
[summerospp]add fluentbit mqtt plugin

### PR DESCRIPTION
**Pull request by sjliu1**
### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by wenchajun — APPROVED** [review]

File: apis/fluentbit/v1alpha2/clusterinput_types.go [hunk 1]
```diff
@@ -59,6 +59,8 @@ type InputSpec struct {
 	OpenTelemetry *input.OpenTelemetry `json:"openTelemetry,omitempty"`
 	// HTTP defines forward  input plugin configuration
```
Comment (benjaminhuo) [inline apis/fluentbit/v1alpha2/clusterinput_types.go:60]:
```suggestion
	// HTTP defines the HTTP input plugin
```

Comment (benjaminhuo) [reply apis/fluentbit/v1alpha2/clusterinput_types.go:60]: // OpenTelemetry defines the OpenTelemetry input plugin

Comment (sjliu1) [reply apis/fluentbit/v1alpha2/clusterinput_types.go]: Fixed, thanks!

File: apis/fluentbit/v1alpha2/clusterinput_types.go [hunk 2]
```diff
@@ -59,6 +59,8 @@ type InputSpec struct {
...
 	HTTP *input.HTTP `json:"http,omitempty"`
+	// MQTT defines forward  input plugin configuration
```
[the remark this answers is not in the document]
Comment (sjliu1) [reply apis/fluentbit/v1alpha2/clusterinput_types.go]: Fixed, thanks!

File: docs/fluentbit.md [hunk 3]
```diff
@@ -420,7 +420,8 @@ InputSpec defines the desired state of ClusterInput
 | customPlugin | CustomPlugin defines Custom Input configuration. | *custom.CustomPlugin |
 | forward | Forward defines forward  input plugin configuration | *[input.Forward](plugins/input/forward.md) |
 | openTelemetry | OpenTelemetry defines forward  input plugin configuration | *[input.OpenTelemetry](plugins/input/opentelemetry.md) |
-| http | HTTP defines forward  input plugin configuration | *[input.OpenTelemetry](plugins/input/opentelemetry.md) |
+| http | HTTP defines forward  input plugin configuration | *[input.HTTP](plugins/input/http.md) |
```
Comment (benjaminhuo) [inline docs/fluentbit.md:423]: Docs not updated

Comment (sjliu1) [reply docs/fluentbit.md]: Done it, thanks!

**Review by wenchajun — APPROVED** [review]

**Pull request merged**